strawberry/schema/schema_converter.py

```diff
@@ -426,6 +427,22 @@ def from_interface(
             assert isinstance(graphql_interface, GraphQLInterfaceType)  # For mypy
             return graphql_interface
 
+        def _get_resolve_type():
+            if interface.resolve_type:
+                return interface.resolve_type
+
+            def resolve_type(
+                obj: Any, info: GraphQLResolveInfo, abstract_type: GraphQLAbstractType
+            ) -> Union[Awaitable[Optional[str]], str, None]:
+                if isinstance(obj, interface.origin):
+                    return obj._type_definition.name
```

Comment:
this should use __strawberry_definition__ (or the getter function)

Reply:
Right! Good catch :)